File: core/src/org/rexellentgames/dungeon/util/CollisionHelper.java
package org.rexellentgames.dungeon.util;

public class CollisionHelper {
	public static boolean check(int px, int py, int x, int y, int w, int h) {
		return px >= x && py >= y && px <= x + w && py <= y + h;
	}
	
	public static boolean check(float x, float y, float w, float h, float x1, float y1, float w1, float h1) {
		return (x < x1 + w1 &&
			x + w > x1 &&
			y < y1 + h1 &&
			y + h > y1);
	}
}

File: BurningKnight/Util/CollisionHelper.cs
namespace BurningKnight.Util
{
	public static class CollisionHelper
	{
		public static bool Check(float l1, float t1, float r1, float b1, float l2, float t2, float r2, float b2)
		{
			return r1 >= l2 &&
        b1 >= t2 &&
        l1 <= r2 &&
        t1 <= b2;
		} 
	}
}